home_task_1/src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Usually we use `.jsx` extension of files for react components

it would be `home_task_1/src/App.jsx`

Reply:
Modified